views/Toast.js

```diff
@@ -1,26 +1,27 @@
-module.exports = Object.create( Object.assign( {}, require('../../../client/js/views/__proto__'), {
+const View = require('view')
 
-    ToastMessage: require('./ToastMessage'),
+module.exports = class Toast extends View {
 
-    name: 'Toast',
+    ToastMessage = require('./ToastMessage')
+
+    name = 'Toast'
```

Comment:
Do you recall why we need name?

Reply:
`name` is generally passed in via the factory, and we have been creating `Toast` directly in the client router, so it needs to have the property added to the view. Not sure if you want to continue doing it this way.